### DISCUSSION TITLE
PrescribedBurnEscrow

### DISCUSSION THREAD

**Discussion by ghost** [XRPLF/XRPL-Standards#140] [XLS Ideas (pre standard proposal)]
Parenting three Saint Bernards is no easy task. Three generations and 450 pounds of dogs is as clumsy and chaotic as you can probably imagine.  Usually once a week we walk along Laurel Ridge State Park, Pennsylvania so that they can run around and feel free. Since we live in a $25,000 house confined in the middle of one of the poorest neighborhoods, Moxham,  in the Flood City of Johnstown, with my life savings tied up in XRP (https://xrp.firmfoundation.io/.well-known/xrp-ledger.toml), we sacrifice short-term discomfort for the next Global Financial System. A few weeks leading into Spring, Laurel Ridge was shut down due to a “Prescribed Burn”. The language was comical considering the timing. I had quit my Software Engineering job at BAE Systems because of a forced Prescribed jab. I served 15 years and in the end I was a weed that needed to be burned to protect the forest. Looking for a sanctuary from the evils of corporate America, I turned to crypto, and chose XRP since it was in a similar position with the SEC. I predicted XRP's win within a few days, see my TikTok if you got down that rabbit hole. XRP’s instant payments, low TX costs, private, immutable and decentralized nature, all were befitting of the next world reserve currency. However, I'm unemployed and my savings are dwindling and the dogs are hungry. I wish to begin a formal conversation about the XRP escrow and its Prescribed Burn. Such a process can be prescribed simultaneously with liquidity to ensure retail investors aren’t weeded out.

**Comment by xVet** [comment]
Welcome to the XRPL community!
